includes/admin/class-wp-job-manager-settings.php

```diff
@@ -369,6 +378,13 @@ protected function init_settings() {
 							'desc'  => __( 'Select the page where you\'ve used the [jobs] shortcode. This lets the plugin know the location of the job listings page.', 'wp-job-manager' ),
 							'type'  => 'page',
 						],
+						[
+							'name'  => 'job_manager_terms_and_conditions_page_id',
+							'std'   => '',
+							'label' => __( 'Terms and Conditions Page', 'wp-job-manager' ),
+							'desc'  => __( 'Select the page that you want to link when agreement checkbox is enabled. See "Agreement checkbox" setting in "Job Submission" tab.', 'wp-job-manager' ),
```

Comment:
Just noting that we'll need to update the label here if we do end up changing it above.

Reply:
https://github.com/Automattic/WP-Job-Manager/pull/2155/commits/ea513f367b0e59288a60c7272c5b7ea9d450f545